src/objective-c/tests/Tests.xcodeproj/project.pbxproj

```diff
@@ -367,12 +367,15 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 5E8A5DAE1D3840B4000F8BC4 /* Build configuration list for PBXNativeTarget "CoreCronetEnd2EndTests" */;
 			buildPhases = (
+				19087EB522BF2CB904F217C6 /* 📦 Check Pods Manifest.lock */,
```

Comment:
I did not mean to add this file into commit...sorry; gonna remove it. But I am using v1.0.0 right now.